api/src/main/java/org/pmiops/workbench/config/CommonConfig.java

```diff
@@ -0,0 +1,25 @@
+package org.pmiops.workbench.config;
+
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.apache.ApacheHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Created by danrodney on 7/13/17.
+ */
+@Configuration
+public class CommonConfig {
+
+  @Bean
```

Comment:
I take it this is all Spring's equivalent of Guice? (Seems fine/good, just checking if my mental map is right.)